lambda/src/main/scala/pricemigrationengine/handlers/SubscriptionIdUploadHandler.scala

```diff
@@ -12,7 +12,7 @@ import scala.jdk.CollectionConverters._
 
 object SubscriptionIdUploadHandler extends CohortHandler {
 
-  private val csvFormat = CSVFormat.Builder.create().setHeader().setSkipHeaderRecord(true).build()
+  private val csvFormat = CSVFormat.Builder.create().setHeader().build()
```

Comment:
I wonder if you need to get rid of `setHeader` as well.  Perhaps a test would help.